apps/site/src/components/PageLanding/CommunityCard.tsx

```diff
@@ -1,12 +1,18 @@
-import Image from 'next/image';
+import Image from 'next/future/image';
 
 interface Props {
   name: string;
   image: string;
   url: string;
+  unoptimized?: boolean;
 }
 
-export default function CommunityCard({ name, image, url }: Props) {
+export default function CommunityCard({
+  name,
+  image,
+  url,
+  unoptimized = false,
```

Comment:
```suggestion
  unoptimized,
```

so that we follow the default value of `next/future/image`'s default value for `unoptimized`.